android/src/main/AndroidManifest.xml

```diff
@@ -1,5 +1,12 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     >
+    <application>
```

Comment:
same here, you should be able to define this in your app layer, not in the plugin.